IHP/ModelSupport.hs

```diff
@@ -802,6 +797,27 @@ didChange field record = didTouchField && didChangeField
             |> fromMaybe (error "didChange failed to retrieve originalDatabaseRecord")
             |> getField @fieldName
 
+-- | Returns 'True' if 'set' was called on that field
+--
+-- __Example:__ Returns 'False' for freshly fetched records
+--
+-- >>> let projectId = "227fbba3-0578-4eb8-807d-b9b692c3644f" :: Id Project
+-- >>> project <- fetch projectId
+-- >>> didTouchField #name project
+-- False
+--
+-- __Example:__ Returns 'True' after setting a field
+--
+-- >>> let projectId = "227fbba3-0578-4eb8-807d-b9b692c3644f" :: Id Project
+-- >>> project <- fetch projectId
+-- >>> project |> set #name project.name |> didChange #name
+-- True
+--
+didTouchField :: forall fieldName fieldValue record. (KnownSymbol fieldName, HasField fieldName record fieldValue, HasField "meta" record MetaBag, Eq fieldValue, Typeable record) => Proxy fieldName -> record -> Bool
+didTouchField field record =
+    record.meta.touchedFields
+    |> includes (cs $! symbolVal field)
```

Comment:
Why is `$!` required here?

Reply:
it's a strict version of `$`, I used it here because I felt like it could improve performance here.

I think we can actually use [`IHP.HaskellSupport.symbolToText`](https://github.com/digitallyinduced/ihp/blob/master/IHP/HaskellSupport.hs#L374) here:

```haskell
|> includes (symbolToText @fieldName)
```

This is even a bit shorter

Reply:
Sure, let me re-roll the PR.